src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -209,21 +209,21 @@ def create_chart_from_prometheus_query(
         label = get_target_name(series)
 
         if not label:
-            label = "\n".join([v for (key, v) in series.metric.items() if key != "job"])
+            label = "\n".join([v for (key, v) in series["metric"].items() if key != "job"])
 
         # If the label is empty, try to take it from the additional_label_factory
         if label == "" and chart_label_factory is not None:
             label = chart_label_factory(i)
 
         values = []
-        for index in range(len(series.values)):
-            timestamp = series.timestamps[index]
-            value = round(float(series.values[index]), FLOAT_PRECISION_LIMIT)
+        for index in range(len(series["values"])):
```

Comment:
I think the type of `series` is ambiguous
It's a `dict` or `PrometheusSeries`

Reply:
typing is unclear it is a dict, fixing